mount.go

```diff
@@ -95,9 +96,14 @@ func mount(source, destination, fsType string, flags int, options string) error
 		return fmt.Errorf("need mount FS type")
 	}
 
-	if fsType != type9pFs {
-		var err error
-
+	var err error
+	switch fsType {
+	case typeTmpFs, type9pFs:
+		if err := createDestinationDir(destination); err != nil {
```

Comment:
@harche Are you not already creating the mount destination directory in the "ephemeral" storage handler?

Reply:
Yes, I will update the PR. Thanks.